libs/dotnet-sdk/FeatureBoardHttpClient.cs

```diff
@@ -33,14 +37,34 @@ public FeatureBoardHttpClient(HttpClient httpClient, LastETagProvider lastModifi
     if (null != eTag)
       request.Headers.IfNoneMatch.Add(eTag);
 
+    var retryAfter = _retryAfter();
+    if (retryAfter != null && retryAfter.Date > DateTimeOffset.Now)
+    {
+      // Do not call API
+      _logger.LogWarning("Failed to get latest flags. Service returned error 429 (Too Many Requests). Retry after: {retryAfter}", retryAfter.Date.Value.ToString());
+      return false;
+    }
     using var response = await _httpClient.SendAsync(request, cancellationToken);
 
+    retryAfter = updateRetryAfterRef(response.Headers.RetryAfter);
+
     switch (response.StatusCode)
     {
       case HttpStatusCode.NotModified:
         _logger.LogDebug("No changes");
         return false;
 
+      //HttpStatusCode.TooManyRequests not defined in .NET Standards 2.0 
+      case (HttpStatusCode)429:
```

Comment:
Maybe we should use case HttpStatusCode.TooManyRequests just to read a bit nicer?

Reply:
HttpStatusCode.TooManyRequests not defined in .NET Standards 2.0